### ISSUE TITLE
am I doing it wrong ?

### ISSUE THREAD

**Issue by xialisuper** [Milad-Akarie/auto_route_library#15]
[INFO] Starting Build

[INFO] Updating asset graph...
[INFO] Updating asset graph completed, took 1ms

[INFO] Running build...
[INFO] Running build completed, took 7ms

[INFO] Caching finalized dependency graph...
[INFO] Caching finalized dependency graph completed, took 14ms

[INFO] Succeeded after 23ms with 0 outputs (0 actions)

and it stops here . I got noting file names like Route.dart.

**Comment by Milad-Akarie** [comment]
@xialisuper  can you please show me your route declaration class.

**Comment by xialisuper** [comment]
import 'package:auto_route/auto_route_annotations.dart';
import 'package:route/page_one.dart';
import 'package:route/page_two.dart';

@autoRouter
class $Route {
  PageOne pageOne;
  PageTwo pageTwo; 
}


xialiqundeMacBook-Pro:route xialisuper$ flutter pub run build_runner watch --delete-conflicting-outputs
[INFO] Generating build script...
[INFO] Generating build script completed, took 364ms

[INFO] Creating build script snapshot......
[INFO] Creating build script snapshot... completed, took 9.9s

[INFO] Setting up file watchers...
[INFO] Setting up file watchers completed, took 16ms

[INFO] Waiting for all file watchers to be ready...
[INFO] Waiting for all file watchers to be ready completed, took 199ms

[INFO] Initializing inputs
[INFO] Building new asset graph...
[INFO] Building new asset graph completed, took 615ms

[INFO] Checking for unexpected pre-existing outputs....
[INFO] Checking for unexpected pre-existing outputs. completed, took 1ms

[INFO] Running build...
[INFO] Generating SDK summary...
[INFO] 3.1s elapsed, 0/5 actions completed.
[INFO] Generating SDK summary completed, took 3.1s

[INFO] 4.2s elapsed, 3/5 actions completed.
[INFO] 9.3s elapsed, 4/5 actions completed.
[INFO] Running build completed, took 10.7s

[INFO] Caching finalized dependency graph...
[INFO] Caching finalized dependency graph completed, took 33ms

[INFO] Succeeded after 10.7s with 0 outputs (5 actions)

**Comment by Milad-Akarie** [comment]
replace @autorouter with @autoRouter

**Comment by xialisuper** [comment]
dependencies:
  flutter:
    sdk: flutter

  cupertino_icons: ^0.1.2
  auto_route: 

dev_dependencies:
  flutter_test:
    sdk: flutter
  build_runner:
  auto_route_generator:

**Comment by Milad-Akarie** [comment]
Everything looks fine actually
here's what you could do
- Clean the build folder by running
`flutter packages pub run build_runner clean`
- Make sure you save all of your files before running the generator
- Make sure it's 
`@autoRouter not @autorouter`

**Comment by xialisuper** [comment]
[SEVERE] Failed to snapshot build script .dart_tool/build/entrypoint/build.dart.
This is likely caused by a misconfigured builder definition.
[SEVERE] Unable to open file .dart_tool/build/entrypoint/build.dart.snapshot for writing snapshot
pub finished with exit code 78

**Comment by xialisuper** [comment]
I just cleaned my project.
And checked for @autoRouter

**Comment by xialisuper** [comment]
thank you ! finally !
thanks for your time and help !

**Comment by Milad-Akarie** [comment]
That's a bit weird!
try specifying the version names
auto_route: ^0.2.1
auto_route_generator: 0.2.1+3

**Comment by Milad-Akarie** [comment]
@xialisuper  You're welcome.
What was it if I may ask

**Comment by xialisuper** [comment]
I thought it was caused by @autoroute .the 'R'. but it was somehow 'corrected' by github.
I checked cmd+z .It is always '@autoRouter' . 

I got this :
[SEVERE] Failed to snapshot build script .dart_tool/build/entrypoint/build.dart.
This is likely caused by a misconfigured builder definition.
[SEVERE] Unable to open file .dart_tool/build/entrypoint/build.dart.snapshot for writing snapshot
pub finished with exit code 78

then I clean project again .it works .
